src/Services/Subscriptions/SubscriptionProcessorService.cs

```diff
@@ -955,7 +975,7 @@ private static bool IvListMatches(List<string> ivList, PokemonData pokemon)
             }
 
             var matches = ivList?.Contains($"{pokemon.Attack}/{pokemon.Defense}/{pokemon.Stamina}") ?? false;
```

Comment:
Crap, this will probably trigger if `ivList` contains `1-5/15/15` and Pokemon IV string is `5/15/15`. Which is valid but should only return true if checked via below range logic.